### PR TITLE
docs(undp): add steps to migrate from mongodb in README

### PR DESCRIPTION
**Pull request by foochifa**
## Problem

FRM-1280

## Solution

Add steps to replace Mongoose using Prisma or Ottoman

Add steps to preliminarily set-up CockroachDB as MongoDB replacement

### CODE REVIEW

**Comment by foochifa** [comment]
Hi @KenLSM @sebastianwzq, have ported over the write up to our main README, but do let me know if you think it is more suited to be in say `/src/README.md` or any other file locations. Wanted to get this up before tomorrow morning so I could ping UNDP!

File: README.md [hunk 1]
```diff
@@ -217,6 +217,57 @@ Scripts for common tasks in MongoDB can be found [here](docs/MONGODB.md).
 
 Please contact FormSG (support@form.gov.sg) for any details.
 
+## Database Alternatives
+
+### Migrating from Mongoose ODM to Prisma ORM
+
+Currently our FormSG uses Mongoose as the Object-Document Mapping (ODM) to MongoDB. This means that our code is strongly coupled with MongoDB as Mongoose solely supports it.
```
Comment (KenLSM) [inline README.md]: /nit: Remove `Currently our`

`FormSG uses ...` should be sufficient as "we" and "FormSG" are referring to the same thing.

File: README.md [hunk 2]
```diff
@@ -217,6 +217,57 @@ Scripts for common tasks in MongoDB can be found [here](docs/MONGODB.md).
...
+
+In order to use a different database with FormSG we will have to first migrate from Mongoose to other object modelling libraries. One of which is Prisma.
```
Comment (KenLSM) [inline README.md]:
```diff
- FormSG we will have
+ FormSG you will have
```

**Review by KenLSM — APPROVED** [review]
The Table of Content doesn't seem to be reflecting the newly added section on Database Alternatives.

Keeping them at README might be sufficient for now since the section is rather small and self-contained.

P.S. Im going to fix some grammar (we -> you) for consistency,

**Pull request merged**